app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/SearchResultsAdapter.java

```diff
@@ -71,7 +71,7 @@ public void onPos(final SourceContent sourceContent, boolean b) {
                         if (imageList == null || imageList.size() == 0) {
                             holder.previewImageView.setVisibility(View.GONE);
                         } else {
-                            Glide.with(context).load(imageList.get(0))
+                            Glide.with(context.getApplicationContext()).load(imageList.get(0))
```

Comment:
@amitiwary999 getting the application context is definitely a bad idea.
Try using the context attached with the holder's widgets. `holder.reviewImageVIew.getContext()`

If this still gives you issues, you might want to switch to Picasso instead.
It works perfectly fine without any such issues.